### ISSUE TITLE
Rotating platform bug

### ISSUE THREAD

**Issue by vrubleg** [renaudbedard/fez-1.12-issues#123]
Demo video: https://youtu.be/IiI8KfQn414?t=15s

I assume that Gomez have to be rotated with the rotating platform when it walks on this platform.

Not significant, just for the collection.

**Comment by renaudbedard** [comment]
Yep that's a historical issue, been present on all releases of the game. I can't quite recall what was the limitation here.

Unlikely to be fixed for 1.12 but thanks for reporting regardless :)
